composer.json

```diff
@@ -12,13 +12,23 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^5.4 | ^7.0"
+        "php": "^5.4 | ^7.0",
+        "dhii/factory-interface": "0.1-alpha1 | 0.1-alpha2",
+        "dhii/callback-abstract": "0.1-alpha2",
+        "dhii/invocable-interface": "0.1-alpha1",
+        "dhii/invocable-base": "dev-task/initial-classes",
```

Comment:
You can now depend on [`dhii/invocable-base@0.1-alpha1`][dhii/invocable-base@0.1-alpha1].

[dhii/invocable-base@0.1-alpha1]: https://github.com/Dhii/invocable-base/releases/tag/v0.1-alpha1